changelogs/fragments/1268-lambda-execute-arn.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - lambda_execute - Fix error where ``function_arn`` can be passed instead of ``name``
```

Comment:
```suggestion
  - lambda_execute - Fix waiter error where ``function_arn`` is be passed instead of ``name``(https://github.com/ansible-collections/amazon.aws/issues/1268).
```